frontend/src/lib/components/entityDetails/InfoItem.svelte

```diff
@@ -24,10 +25,20 @@ Used to show a label-content pair in a Candidate's basic information.
 ```
 -->
 
-<div class="infoItem grid grid-flow-col grid-cols-[min-content_auto] justify-start gap-md">
+<div class="grid justify-start gap-md {vertical ? 'vertical-grid' : 'horizontal-grid'}">
   <!-- pt-[0.3rem] matches the baselines of the small label and the content text -->
-  <div class="small-label w-min min-w-[7rem] pt-[0.3rem] text-left align-top">{label}</div>
+  <div class="small-label pt-[0.3rem] text-left align-top {vertical ? 'w-max' : 'min-w-[10rem]'}">{label}</div>
   <div class="overflow-hidden align-top">
     <slot />
   </div>
 </div>
+
+<style lang="postcss">
```

Comment:
Even though Tailwind [recommends to avoid using @apply](https://tailwindcss.com/docs/reusing-styles#avoiding-premature-abstraction) just to avoid listing multiple classes. I believe when we do so within a Svelte component (in an isolated manner) - it is justified, as maintaining our own CSS classes only per component is not that cognitively demanding.

Reply:
Yeah, I think it's fine in general to use @apply in situations like this. In this particular one, however, I think it might also be okay to just stash everything within the ternary op so that one could see all styling happening at a glance, but it's nice this way as well.